src/main/scala/de/gccc/jib/SbtJibHelper.scala

```diff
@@ -22,4 +26,17 @@ private[jib] object SbtJibHelper {
     layerBuilder.build()
   }
 
+  def writeJibOutputFiles(targetDirectory: File, container: JibContainer): Unit = {
+    Files.write(targetDirectory.toPath.resolve("jib-image.digest"), container.getDigest.toString.getBytes(UTF_8))
+    Files.write(targetDirectory.toPath.resolve("jib-image.id"), container.getImageId.toString.getBytes(UTF_8))
+    val jsonString =
+      s"""{
+         |   "image": "${container.getTargetImage}",
+         |   "imageId": "${container.getImageId}",
+         |   "imageDigest": "${container.getDigest}",
+         |   "tags": ${container.getTags.asScala.mkString("[\"", "\", \"", "\"]")}
+         |}""".stripMargin
```

Comment:
@schmitch I did this to avoid a JSON library dependency since the output is very simple, but if you'd prefer to use a proper JSON encoder I'm fine with that.

Reply:
I would have preferred to re-use the code from jib, but sadly this is all part of the `jib-plugins-common` module which is not published.

Reply:
reported this: https://github.com/GoogleContainerTools/jib/issues/3578